native/Avalonia.Native/src/OSX/KeyTransform.h

```diff
@@ -1,9 +1,10 @@
 #ifndef keytransform_h
 #define keytransform_h
 #include "common.h"
-#include "key.h"
 #include <map>
 
 extern std::map<int, AvnKey> s_KeyMap;
```

Comment:
curiosity ( and my C++  is rusty), we cannot have this as extern const std::map ?

Reply:
doesnt compile unfortunately.